src/parser.mly

```diff
@@ -61,7 +61,15 @@ cmd:
     LBRACK c = cmd RBRACK
     { CFor (x, x1, x2, 1, c) } ;
   | c1 = cmd c2 = cmd
-    { CSeq (c1, c2) }
```

Comment:
To respond to the question about which case happens: menhir currently resolves this in an arbitrary manner. If you want to make it more obvious, we should change the rule to be `c1 = cmd SEMICOLON c2 = cmd` and remove `SEMICOLON` from other rules. Additionally, we define `%right SEMICOLON` so that only the second case happens.

Reply:
This is done in a better way in https://github.com/cucapra/seashell/commit/6ad10c48ec36235c31b280dc8714827296fa4df5